Britannia/common/buildings/BTWK_special_mines.txt

```diff
@@ -564,9 +564,8 @@ btwk_iron_mines_01 = {
     }
 
     character_modifier = {
-        heavy_infantry_maintenance_cost_mult = -0.02
-        heavy_cavalry_maintenance_cost_mult = -0.02
-        pikemen_maintenance_cost_mult = -0.02
+        army_maintenance_mult = -0.01
+        men_at_arms_recruitment_cost = -0.05
```

Comment:
I think the modifiers are good but the mines could probably give a bigger income bonus, I think

Reply:
All mines in general you reckon, or specific ones? I did scale the income down a quite a bit across the board from vanilla mines, seeing as they were more frequent.

Reply:
Sort of thinking in general - in vanilla even the smallest mines gets 2/3.5/5/6.5, which is several times more than even the gold mine. TBH I think the flat income could be as much as doubled, even taking to account that they're somewhat more common than in vanilla.